docs/Manpage.md

```diff
@@ -1945,6 +1945,9 @@ example, run `export HOMEBREW_NO_INSECURE_REDIRECT=1` rather than just
 
   *Default:* `300`.
 
+- `HOMEBREW_AUTOREMOVE`
+  <br>If set, calls to `brew install`, `brew upgrade`, `brew reinstall` and `brew uninstall` will automatically remove unused formula dependents.
```

Comment:
I thought we were going to test this out for a while first before making it the default. This would make running `brew autoremove` every 30 days the default. Is that what we want?

Reply:
Sorry, what I was intending with this messaging is "if you set HOMEBREW_AUTOREMOVE and not HOMEBREW_NO_INSTALL_CLEANUP: `brew cleanup` will start doing `brew autoremove` periodically".